Modules/Poll/classes/class.ilPollUserTableGUI.php

```diff
@@ -105,9 +105,9 @@ protected function fillRowCSV(ilCSVWriter $a_csv, array $a_set) : void
         $a_csv->addColumn((string) ($a_set["firstname"] ?? ''));
         foreach ($this->answer_ids as $answer_id) {
             if ($a_set["answer" . $answer_id]) {
-                $a_csv->addColumn(true);
+                $a_csv->addColumn('true');// TODO PHP8-REVIEW I am not sure if my fix is correct
```

Comment:
I am not sure if my fix is correct

Reply:
In addColumn() the csv-string is concatenated with the .= operator and thus an implicit string cast is performed. 
So I guess addColumn('1'); should be compatible with previous implementation.